src/services/DynamoDBService.ts

```diff
@@ -131,7 +131,13 @@ export class DynamoDBService {
       ExpressionAttributeValues: filterValues
     };
 
-    return DynamoDBService.client.query(query).promise();
+    console.log('params for getOngoingByStaffId', query);
```

Comment:
the console log stmt needs to be before return stmt otherwise it will not be executed

Reply:
Not sure if the comparison has gone awry, but the log statement is before the call, that return statement has been completely removed.

    console.log('params for getOngoingByStaffId', query);
    try {
      return await this.queryAllData(query);
    } catch (err) {
      console.error('error on getOngoingByStaffId', err);
      throw err;
    }